plugins/CircuitExplorer/CircuitExplorerPlugin.cpp

```diff
@@ -22,8 +22,8 @@
 #include "CircuitExplorerPlugin.h"
 
 #include <brayns/common/Log.h>
+#include <brayns/common/parameters/ParametersManager.h>
```

Comment:
I think I missed it in the last CircuitExplorer update, but seems like this include is not needed